06-func.md

```diff
@@ -52,136 +52,64 @@ Let's try running our function.
-and printing out the value of each part.
-
-~~~ {.python}
-# We'll use temp = 212, the boiling point of water, which was incorrect
-print "212 - 32:", 212 - 32
-~~~
-~~~ {.output}
-212 - 32: 180
-~~~
-
-~~~ {.python}
-print "(212 - 32) * (5/9):", (212 - 32) * (5/9)
-~~~
-~~~ {.output}
-(212 - 32) * (5/9): 0
-~~~
-
-Aha! The problem comes when we multiply by `5/9`.
-This is because `5/9` is actually 0.
-
-~~~ {.python}
-5/9
-~~~
-~~~ {.output}
-0
-~~~
-
-Computers store numbers in one of two ways:
-as [integers](reference.html#integer)
-or as [floating-point numbers](reference.html#floating-point-number) (or floats).
-The first are the numbers we usually count with;
-the second have fractional parts.
-Addition, subtraction and multiplication work on both as we'd expect,
-but division works differently.
-If we divide one integer by another,
-we get the quotient without the remainder:
-
-~~~ {.python}
-print '10/3 is:', 10/3
-~~~
-~~~ {.output}
-10/3 is: 3
-~~~
-
-If either part of the division is a float,
-on the other hand,
-the computer creates a floating-point answer:
 
-~~~ {.python}
-print '10.0/3 is:', 10.0/3
-~~~
-~~~ {.output}
-10.0/3 is: 3.33333333333
-~~~
-
-The computer does this for historical reasons:
-integer operations were much faster on early machines,
-and this behavior is actually useful in a lot of situations.
-It's still confusing,
-though,
-so Python 3 produces a floating-point answer when dividing integers if it needs to.
-We're still using Python 2.7 in this class,
-though,
-so if we want `5/9` to give us the right answer,
-we have to write it as `5.0/9`, `5/9.0`, or some other variation.
-
-Another way to create a floating-point answer
-is to explicitly tell the computer that you desire one.
-This is achieved by [casting](reference.html#typecast) one of the numbers:
-
-~~~ {.python}
-print 'float(10)/3 is:', float(10)/3
-~~~
-~~~ {.output}
-float(10)/3 is: 3.33333333333
-~~~
-
-The advantage to this method is it can be used with existing variables.
-Let's take a look:
-
-~~~ {.python}
-a = 10
-b = 3
-print 'a/b is:', a/b
-print 'float(a)/b is:', float(a)/b
-~~~
-~~~ {.output}
-a/b is: 3
-float(a)/b is: 3.33333333333
-~~~
-
-Let's fix our `fahr_to_kelvin` function with this new knowledge:
-
-~~~ {.python}
-def fahr_to_kelvin(temp):
-    return ((temp - 32) * (5.0/9.0)) + 273.15
-
-print 'freezing point of water:', fahr_to_kelvin(32)
-print 'boiling point of water:', fahr_to_kelvin(212)
-~~~
-~~~ {.output}
-freezing point of water: 273.15
-boiling point of water: 373.15
-~~~
+> ## Integer division {.callout}
+>
+> We are using Python3 and integer division always return float points.
+>
+> ~~~ {.python}
+> $ python -c "print(5/9)"
+> ~~~
+> ~~~ {.output}
+> 0.5555555555555556
+> ~~~
+>
+> **Unfortunately**, in Python2 this isn't the case.
+>
+> ~~~ {.bash}
+> $ python -c "print(5/9)"
+> ~~~
+> ~~~ {.output}
+> 0
+> ~~~
+>
+> If you are using Python2 and want a float point as return of division
```

Comment:
float -> floating. (Though is "floating point" actually defined in this lesson at all?)